assets/bingo/lang/pl_pl.json

```
@@ -0,0 +1,411 @@
+	
+	"bingo.got_bingo.double.col_row": "[%s] %s zdobył podwójne bingo w kolumnie %s i wierszu %s!",
+	"bingo.got_bingo.double.col_diag.top_left": "[%s] %s zdobył podwójne bingo w kolumnie %s i na przekątnej od lewej górnej do prawej dolnej!",
+	"bingo.got_bingo.double.col_diag.bottom_left": "[%s] %s zdobył podwójne bingo w kolumnie %s i na przekątnej od lewej dolnej do prawej górnej!",
+	"bingo.got_bingo.double.row_diag.top_left": "[%s] %s zdobył podwójne bingo w wierszu %s i na przekątnej od lewej górnej do prawej dolnej!",
+	"bingo.got_bingo.double.row_diag.bottom_left": "[%s] %s zdobył podwójne bingo w wierszu %s i na przekątnej od lewej dolnej do prawej górnej!",
+	"bingo.got_bingo.double.diags": "[%s] %s zdobył podwójne bingo na obu przekątnych!",
+	
+	"bingo.got_bingo.triple.top_left": "[%s] %s zdobył potrójne bingo w kolumnie %s, wierszu %s i na przekątnej od lewej górnej do prawej dolnej!",
+	"bingo.got_bingo.triple.bottom_left": "[%s] %s zdobył potrójne bingo w kolumnie %s, wierszu %s i na przekątnej od lewej dolnej do prawej górnej!",
+	"bingo.got_bingo.triple.col_diags": "[%s] %s zdobył potrójne bingo w kolumnie %s i na obu przekątnych!",
+	"bingo.got_bingo.triple.row_diags": "[%s] %s zdobył potrójne bingo w wierszu %s i na obu przekątnych!",
+	
+	"bingo.got_bingo.quadruple": "[%s] %s zdobył poczwórne bingo!",
+
+	"bingo.got_sniper_bingo.row": "[%s] %s zdobył sniper-bingo w wierszu %s!",
+	"bingo.got_sniper_bingo.col": "[%s] %s zdobył sniper-bingo w kolumnie %s!",
+	"bingo.got_sniper_bingo.diag.top_left": "[%s] %s zdobył sniper-bingo na przekątnej od lewej górnej do prawej dolnej!",
+	"bingo.got_sniper_bingo.diag.bottom_left": "[%s] %s zdobył sniper-bingo na przekątnej od lewej dolnej do prawej górnej!",
+
+	"bingo.got_sniper_bingo.double.col_row": "[%s] %s zdobył podwójne sniper-bingo w kolumnie %s i wierszu %s!",
+	"bingo.got_sniper_bingo.double.col_diag.top_left": "[%s] %s zdobył podwójne sniper-bingo w kolumnie %s i na przekątnej od lewej górnej do prawej dolnej!",
+	"bingo.got_sniper_bingo.double.col_diag.bottom_left": "[%s] %s zdobył podwójne sniper-bingo w kolumnie %s i na przekątnej od lewej dolnej do prawej górnej!",
+	"bingo.got_sniper_bingo.double.row_diag.top_left": "[%s] %s zdobył podwójne sniper-bingo w wierszu %s i na przekątnej od lewej górnej do prawej dolnej!",
+	"bingo.got_sniper_bingo.double.row_diag.bottom_left": "[%s] %s zdobył podwójne sniper-bingo w wierszu %s i na przekątnej od lewej dolnej do prawej górnej!",
+	"bingo.got_sniper_bingo.double.diags": "[%s] %s zdobył podwójne sniper-bingo na obu przekątnych!",
+
+	"bingo.got_sniper_bingo.triple.top_left": "[%s] %s zdobył potrójne sniper-bingo w kolumnie %s, wierszu %s i na przekątnej od lewej górnej do prawej dolnej!",
+	"bingo.got_sniper_bingo.triple.bottom_left": "[%s] %s zdobył potrójne sniper-bingo w kolumnie %s, row %s i na przekątnej od lewej dolnej do prawej górnej!",
```

```suggestion
	"bingo.got_sniper_bingo.triple.bottom_left": "[%s] %s zdobył potrójne sniper-bingo w kolumnie %s, wierszu %s i na przekątnej od lewej dolnej do prawej górnej!",
```
Missed to translate "row", used same translation as previous entry